src/tools/kata-ctl/src/utils.rs

```diff
@@ -62,14 +62,15 @@ mod tests {
         let dir = tempdir().unwrap();
         let file_path = dir.path().join("proc-version");
         let path = file_path.clone();
-        let mut file = fs::File::create(file_path).unwrap();
```

Comment:
In which case does the creating of file may fail?

Reply:
Currently I have no idea. But from the CI's console output, this is where this test normally fails.

Reply:
After this patch, the test will test nothing in case creating a temp file fails.
Do you have any logs?

Reply:
Unfortunately there was no logs, it only said the unwrap causes the panic.
Currently, I only see this operation fails on s390x and arm, but I have no clue about the reasons.

Reply:
@Ji-Xinyou I think this may only fail in GitHub actions, we have some similar errors:

- https://github.com/kata-containers/kata-containers/issues/4103
- https://github.com/kata-containers/kata-containers/issues/4248
- https://github.com/kata-containers/kata-containers/issues/5737

Reply:
It is rlly weird...... It seems like it happens randomly